sw/airborne/subsystems/navigation/common_flight_plan.h

```diff
@@ -44,8 +44,8 @@ void nav_goto_block(uint8_t block_id);
 #define InitStage() nav_init_stage();
 
 #define Block(x) case x: nav_block=x;
-#define NextBlock() { nav_block++; nav_init_block(); }
-#define GotoBlock(b) { nav_block=b; nav_init_block(); }
+#define NextBlock() nav_goto_block(nav_block + 1)
```

Comment:
Are we checking here for overflow in the number of blocks? E.g. I am in the last block and will want to go to the next block?